### PR TITLE
update notification settings

### PR DESCRIPTION
**Pull request by lizokm**
Reflect new notification settings.

### CODE REVIEW

File: src/docs/product/alerts/notifications/notification-settings.mdx [hunk 1]
```diff
@@ -1,88 +1,28 @@
 ---
 title: Personal Notification Settings
 sidebar_order:
-description: "Learn how to manage the notifications that Sentry sends you."
+description: "Learn how to manage my personal notifications settings."
```
Comment (lizokm) [inline src/docs/product/alerts/notifications/notification-settings.mdx:4]:
```suggestion
description: "Learn how to manage your personal notifications settings."
```

File: src/docs/product/alerts/notifications/notification-settings.mdx [hunk 2]
```diff
@@ -1,88 +1,28 @@
...
 ---
 
-You can adjust your personal [workflow](#workflow) and [deploy](#deploy) notifications, as well as your personal issue alert settings, in your [account settings](https://sentry.io/settings/account/notifications/). Manage your notifications by navigating to **User Settings > Notifications**. You **cannot configure** quota notifications.
+You can fine-tune your personal notification settings for things like [workflow](#workflow), [deploy](#deploy), issue alert, and [spike protection](/product/accounts/quotas/spike-protection/) by turning them on or off, specifying how you'd like to be notified (via email, Slack, or both), and more. Users who **aren't** on an [Enterprise](https://sentry.io/pricing/) plan can also configure their quota notifications.
```
Comment (lizokm) [inline src/docs/product/alerts/notifications/notification-settings.mdx:7]:
```suggestion
You can update your personal notification settings for things like [workflow](/product/alerts/notifications/), deploy, issue alert, and [spike protection](/product/accounts/quotas/spike-protection/) by turning them on or off, specifying how you'd like to be notified (via email, Slack, or both), and more. Users who **aren't** on an [Enterprise](https://sentry.io/pricing/) plan can also configure their quota notifications.
```

File: src/docs/product/alerts/notifications/notification-settings.mdx [hunk 3]
```diff
@@ -1,88 +1,28 @@
...
-
-## My Activity
-
-Use the toggles to control whether or not you receive notifications about:
-
-- Your actions while using [sentry.io](https://sentry.io)
-- Any changes on unclaimed issues you've resolved
+- **Issue Alerts:** Notifications sent from alert rules that your team's set up. (You'll still get any notifications that have been set up to be sent directly to you.)
+- **Issue Workflow:** Changes in issue assignment, resolution status, and comments.
+- **Deplys:** Release, environment, and commit overviews.
```
Comment (lizokm) [inline src/docs/product/alerts/notifications/notification-settings.mdx:21]:
```suggestion
- **Deploys:** Release, environment, and commit overviews.
```

File: src/docs/product/alerts/notifications/notification-settings.mdx [hunk 4]
```diff
@@ -1,88 +1,28 @@
...
-- Send to Email and Slack
-
-[Slack](/product/integrations/notification-incidents/slack/) is only available as a delivery method if your organization has the integration installed and your [Slack identity is linked to your Sentry account](/product/integrations/notification-incidents/slack/#linking-your-slack-and-sentry-accounts).
-
-## Workflow
-
-The global settings for workflow notifications are:
-
-- Always
-- Only On Issues I Subscribe To
```
Comment (scefali) [inline src/docs/product/alerts/notifications/notification-settings.mdx:36]: @lizokm why did we remove this bit?

Comment (lizokm) [reply src/docs/product/alerts/notifications/notification-settings.mdx:36]: This functionality is clear if you go to the product and I don't think we add anything in docs by explaining something that's already easy to understand in the product itself. Open to differing opinions :)

Comment (scefali) [reply src/docs/product/alerts/notifications/notification-settings.mdx:36]: @lizokm I think a user might be confused about what it means to be subscribed to an issue, I'm not sure how intuitive that is

Comment (snigdhas) [reply src/docs/product/alerts/notifications/notification-settings.mdx:36]: Agreed that subscriptions are confusing, but we don't really explain what it means to be subscribed in this section?

**Review by snigdhas — APPROVED** [review]

**Review by scefali — APPROVED** [review]

File: src/docs/product/alerts/notifications/notification-settings.mdx (hunk 4 above)
Comment (lizokm) [reply src/docs/product/alerts/notifications/notification-settings.mdx:36]: Ah, I see what you mean. We can just add this section from the main notification page: You receive workflow notifications when you’re subscribed to an issue, and you subscribe to issues by:

Clicking the subscribe bell icon on an issue
Involvement in a commit related to the issue
Commenting on or bookmarking the issue
Mention of you or your team in the issue
You or your team being assigned to the issue
Archiving an issue until it escalates

File: src/docs/product/alerts/notifications/notification-settings.mdx [hunk 5]
```diff
@@ -1,88 +1,28 @@
...
+![Issue alert notification page](issue-alert-notification.png)
 
-<Note>
+To make updates, go to your [user settings](https://sentry.io/settings/account/notifications/), select "Notifications", then click on the notification type you want to update.
 
-This setting does not affect alerts that are configured to send to your email explicitly.
+![Issue alert notification detail page](issue-alert-notification-detail.png)
 
-</Note>
+## Notification Types
```
Comment (lizokm) [inline src/docs/product/alerts/notifications/notification-settings.mdx:15]:
```suggestion
## How to Subscribe to an Issue

You receive workflow notifications once you’ve subscribed to an issue in one of the ways described below:

- By clicking the subscribe "bell icon" on an issue
- By taking part in a commit that's related to the issue
- By commenting on or bookmarking the issue
- By archiving an issue until it escalates
- If you or your team is mentioned in the issue
- If you or your team is assigned to the issue

These notifications may have some overlap with [alerts](/product/alerts/) that are configured for a project.

## Notification Types
```

**Review by snigdhas — APPROVED** [review]

**Pull request merged**